src/datahike/api.cljc

```diff
@@ -11,71 +11,76 @@
               [java.util Date])))
 
 (def
-  ^{:arglists '([uri])
-    :doc
-              "Connects to a datahike database via URI or configuration. URI contains storage backend type
-            and additional information for backends like database name, credentials, or
-            location. Refer to the store project in the examples folder or the documentation
-            in the config markdown file in the doc folder.
+  ^{:arglists '([] [config])
+    :doc "Connects to a datahike database via configuration. For more information on the configuration refer to the [docs](https://github.com/replikativ/datahike/blob/master/doc/config.md).
 
-            Usage:
+The configuration for a connection is a subset of the database configuration with only the store necessary: `:store`.
```

Comment:
instead of database configuration maybe 'Datahike configuration'. I had to think about it because I confused database/datahike/postgres etc. Just a suggestion.